eng/build/Engineering.props

```diff
@@ -0,0 +1,37 @@
+<Project>
+
+  <!-- Additional paths -->
+  <PropertyGroup>
+    <WorkersProps>$(EngBuildRoot)Workers.props</WorkersProps>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Nuget audit as warnings only, even in TreatWarningsAsErrors. -->
+    <!-- Except for in CI, critical will fail the build. -->
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);NU1901;NU1902;NU1903</WarningsNotAsErrors>
+    <WarningsNotAsErrors Condition="'$(CI)' == 'false'">$(WarningsNotAsErrors)NU1904</WarningsNotAsErrors>
+    <WarningsAsErrors Condition="'$(CI)' == 'true'">$(WarningsAsErrors)NU1904</WarningsAsErrors>
+    <NuGetAuditLevel>moderate</NuGetAuditLevel> <!-- warn on moderate severity only. -->
+    <NuGetAuditMode>all</NuGetAuditMode> <!-- audit transitive dependencies. -->
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <AssemblyOriginatorKeyFile>$(EngResourceRoot)key.snk</AssemblyOriginatorKeyFile>
+    <CodeAnalysisRuleSet>$(RepoRoot)src.ruleset</CodeAnalysisRuleSet>
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
+    <DebugType>embedded</DebugType>
+    <GenerateRuntimeConfigDevFile>true</GenerateRuntimeConfigDevFile><!-- https://github.com/dotnet/runtime/issues/54684 -->
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="$(RepoRoot)stylecop.json" Link="stylecop.json" />
+  </ItemGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)Release.props" />
+
+  <!-- Uncomment below to run with placeholder simulation enabled. -->
+  <!-- <Import Project="$(MSBuildThisFileDirectory)PlaceholderSimulation.props" /> -->
```

Comment:
@kshyju want to highlight this change for you. The additional configurations were causing pain points with this refactor so I went with an alternative approach: uncomment this line when you want to test placeholder simulation. The file has a safeguard to fail the build in CI if it is included uncommented in a PR.

If you want a different approach let me know. Another approach is to set an environment variable before build / before opening VS.

Reply:
I would much prefer the environment variable approach over comment/uncommenting code. Seems like something easy to forget, then you have to wait for the CI to start before you realize you forgot to uncomment it etc. etc.

Reply:
@Shyju the property/env variable approach sounds more suitable then. 

1. For publish it will be done via:  `dotnet publish -c release -p:PlaceholderSimulation=true`
2. For local dev, you can set `$env:PlaceholderSimulation = true` before launching VS or VSCode.

Is that acceptable?

Reply:
That it certainly better than comment/uncomment approach. I am curious to learn about the issues you ran into while keeping the additional configurations.

Reply:
The issues are with the config mapping in the solution file. VS kept complaining about invalid config mapping. This wasn't the first time that config mapping got in the way of MSBuild refactors, so I opted to remove the config entirely.